lib/commands/proxy-helper.js

```diff
@@ -1,21 +1,29 @@
 import { errorFromCode } from 'appium-base-driver';
 import log from '../logger';
+import _ from 'lodash';
 
+const SUPPORTED_METHODS = ['POST', 'GET', 'DELETE'];
 
 let helpers = {}, extensions = {};
 
 helpers.proxyCommand = async function (endpoint, method, body) {
   if (!endpoint) {
     log.errorAndThrow('Proxying requires an endpoint');
-  } else if (method !== 'POST' && method !== 'GET' && method !== 'DELETE') {
-    log.errorAndThrow('Proxying only works for GET, POST or DELETE requests');
+  } else if (!_.find(SUPPORTED_METHODS, method)) {
```

Comment:
Testing this out, I don't think this is how `_.find` works. `_.find(SUPPORTED_METHODS, (el) => el === method)` would work but is heavy for the purposes here. `SUPPORTED_METHODS.indexOf(method) === -1` would suffice.